src/porepy/models/contact_mechanics_biot_model.py

```diff
@@ -774,7 +791,10 @@ def _assign_discretizations(self) -> None:
             )
 
             # Accumulation term on the fractures.
-            frac_vol = np.hstack([g.cell_volumes for g in g_frac])
+            if len(g_frac) > 0:
+                frac_vol = np.hstack([g.cell_volumes for g in g_frac])
+            else:
+                frac_vol = np.array([])
```

Comment:
Could this be a MassMatrix?

Reply:
Yes, I think you are right. We should take of that when reworking that model, similar to what you do with pure ContactMechanics for the moment.